### PR TITLE
Upgrading to babel 7

### LINKED ISSUE #25: Update to babel 7
Wait after it will be released. Now in beta.

### PR DESCRIPTION
**Pull request by tommmyy**
Closes #25 and #67

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,14 +56,13 @@
   ],
   "scripts": {
     "example:basic": "cd boilerplates/react-union-boilerplate-basic && yarn start",
-    "lint": "eslint \"packages/*.js\" \"packages/**/*.js\" \"boilerplates/**/*.js\"",
+    "lint": "eslint \"packages/*.js\" \"packages/**/*.js\" \"boilerplates/**/*.js\"^7.0.0",
```
Comment (wafflepie) [inline package.json]: Is this a mistake? It looks really weird.

Comment (tommmyy) [reply package.json]: yes, it is

**Comment by wafflepie** [comment]
Great job!

**Comment by tommmyy** [comment]
@wafflepie @aizerin @jamescoq  Guys, it is green

**Pull request merged**